java/spacewalk-java.changes

```diff
@@ -1,3 +1,6 @@
+  * FIX ISE and NPE when an image profile / store is deleted
```

Comment:
I think we need to start with a dash "-" instead of a start "*".

Reply:
I am also not sure if a customer understand "ISE" and "NPE". Maybe are more generic and say just "error".